package.json

```diff
@@ -14,58 +14,58 @@
     "e2e": "ng e2e"
   },
   "dependencies": {
-    "@angular/animations": "15.0.4",
-    "@angular/cdk": "15.0.3",
-    "@angular/common": "15.0.4",
-    "@angular/compiler": "15.0.4",
-    "@angular/core": "15.0.4",
-    "@angular/forms": "15.0.4",
-    "@angular/platform-browser": "15.0.4",
-    "@angular/platform-browser-dynamic": "15.0.4",
-    "@angular/router": "15.0.3",
-    "@ngx-translate/core": "14.0.0",
-    "@ngx-translate/http-loader": "^7.0.0",
+    "@angular/animations": "16.2.7",
+    "@angular/cdk": "16.2.7",
+    "@angular/common": "16.2.7",
+    "@angular/compiler": "16.2.7",
+    "@angular/core": "16.2.7",
+    "@angular/forms": "16.2.7",
+    "@angular/platform-browser": "16.2.7",
+    "@angular/platform-browser-dynamic": "16.2.7",
+    "@angular/router": "16.2.7",
+    "@ngx-translate/core": "15.0.0",
+    "@ngx-translate/http-loader": "8.0.0",
     "ag-grid-angular": "^28.2.1",
     "ag-grid-community": "^28.2.1",
-    "angular-split": "^4.0.0",
+    "angular-split": "16.2.1",
     "bootstrap": "^4.5.2",
-    "core-js": "^2.5.6",
-    "date-fns": "^2.15.0",
-    "interactjs": "^1.10.17",
+    "core-js": "3.32.2",
+    "date-fns": "2.30.0",
+    "interactjs": "^1.10.19",
     "jquery": "^3.5.1",
     "mobile-drag-drop": "^2.3.0-rc.2",
-    "pako": "^1.0.6",
+    "pako": "^1.0.11",
     "popper.js": "^1.16.1",
     "primeicons": "^6.0.1",
     "primeng": "^14.1.2",
```

Comment:
PrimeNG and primeicons are usually updated as part of the Angular update

Reply:
Updated